modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,205 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer (ALB) Operator
+Users that wish to install the AWS Load Balancer (ALB) Operator can do so as long as the following prerequisites are met:
+
+.Prerequisites
+
+* An existing {product-title} cluster with BYO-VPC installed in STS mode
+* Cluster-admin access
+* Access to modify the VPC and subnets of the created {product-title} cluster
+// * AWS account credentials
+* Configured {product-title} CLI
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+[NOTE]
+====
+When installing for use in an AWS Local Zone, the Local Zone is enabled for the account and the  Application Load Balancer (ALB) Operator is an available feature within the Local Zone. Additionally, the ability to use Amazon ELBv2 with {product-title} is subject to the availability of Application Load Balancers (ALB) and Network Load Balancers (NLB) in the AWS Availability Zones and Local Zones.
+====
+[IMPORTANT]
+====
+The Application Load Balancer (ALB) Operator supports creating Network Load Balancers (NLBs) on {product-title} clusters in either single-AZ or multi-AZ configurations. Application Load Balancers (ALBs) are supported in multi-AZ {product-title} clusters only, because Application Load Balancers (ALBs) require a multi-AZ configuration.
+====
+[WARNING]
+====
+The AWS Application Load Balancer (ALB) Operator must only be installed after the successful installation of a {product-title} cluster. The AWS Application Load Balancer (ALB) Operator requires adding tags to cluster VPCs that may interfere with cluster creation.
+====
+.Procedure
+
+. Identify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS by running the following commands:
+.. Identify the {product-title} cluster INFRA ID:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'Infra ID' OR $oc get infrastructure cluster -o json | jq -r '.status.infrastructureName' ----
```

Comment:
There are two commands can be used to get infra-id, one is `rosa describe...` and one is `oc get ...` , please separate into two lines. And the last `----` should be in new line.

Reply:
> There are two commands can be used to get infra-id, one is `rosa describe...` and one is `oc get ...` , please separate into two lines. And the last `----` should be in new line.

Ack. Will change.